addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1169,7 +1169,11 @@ const UnitUserValueWidget = UserValueWidget.extend({
         if (!params.unit) {
             return isSuperActive;
         }
-        return isSuperActive && this._floatToStr(parseFloat(this._value)) !== '0';
+        return isSuperActive && (
+            this._floatToStr(parseFloat(this._value)) !== '0'
+            // Or is a composite value.
+            || this._value.match(/\d+\s+\d+/)
```

Comment:
By solving the 0px border-radius problem, you also solved the 0px border problem. 🥳 But if you start your list of values with a 0, the border styles and color picker don't show up. That's because your `isActive()` doesn't return a boolean but an array when it matches multiple values. So you could simply write `!!this._value.match(/\d+\s+\d+/)` to solve this.